front/src/common/redux/students/students.actions/studentRegistrationAction.js

```diff
@@ -0,0 +1,5 @@
+import { REGISTER_STUDENT, STUDENT_COMPLETED_REGISTRATION } from './types';
+
+export const studentRegistered = (student) => ({ type: REGISTER_STUDENT, payload: student });
+
+export const registrationCompleted = () => ({ type: STUDENT_COMPLETED_REGISTRATION });
```

Comment:
Action creator name should match action type name

Reply:
Renamed